frontend/app/routes/protected/renew/$id/member-selection.tsx

```diff
@@ -0,0 +1,195 @@
+
+  const csrfToken = String(session.get('csrfToken'));
+  const meta = { title: t('gcweb:meta.title.template', { title: t('protected-renew:member-selection.page-title') }) };
+
+  const applicant = state.applicantInformation;
+  const children = getProtectedChildrenState(state);
+
+  return { csrfToken, meta, applicant, children, editMode: state.editMode };
+}
+
+export async function action({ context: { appContainer, session }, params, request }: ActionFunctionArgs) {
+  const log = getLogger('protected/renew/member-selection');
+
+  const securityHandler = appContainer.get(TYPES.routes.security.SecurityHandler);
+  await securityHandler.validateAuthSession(request);
+
+  const state = loadProtectedRenewState({ params, session });
+
+  const formData = await request.formData();
+  const expectedCsrfToken = String(session.get('csrfToken'));
+  const submittedCsrfToken = String(formData.get('_csrf'));
+
+  if (expectedCsrfToken !== submittedCsrfToken) {
+    log.warn('Invalid CSRF token detected; expected: [%s], submitted: [%s]', expectedCsrfToken, submittedCsrfToken);
+    throw new Response('Invalid CSRF token', { status: 400 });
+  }
+
+  if (state.editMode) {
+    return redirect(getPathById('protected/renew/$id/member-selection', params)); //TODO: Update to 'review and submit' when page is added
+  }
+  return redirect(getPathById('protected/renew/$id/review-adult-information', params));
+}
+
+export default function ProtectedRenewMemberSelection() {
+  const { t } = useTranslation(handle.i18nNamespaces);
+  const { csrfToken, applicant, children, editMode } = useLoaderData<typeof loader>();
+  const params = useParams();
+  const fetcher = useFetcher<typeof action>();
+  const isSubmitting = fetcher.state !== 'idle';
+
+  const applicantName = `${applicant?.firstName} ${applicant?.lastName}`;
+
+  const hasExternalReviews = applicant?.externallyReviewed ?? children.filter((child) => child.externallyReviewed === true).length > 0;
+
+  return (
+    <>
+      <div className="my-6 sm:my-8">
+        <Progress value={81} size="lg" label={t('renew:progress.label')} />
+      </div>
+      <div className="max-w-prose">
+        {hasExternalReviews && (
+          <>
+            <p className="mb-4">{t('protected-renew:member-selection.reviewed')}</p>
+            <ul className="list-disc space-y-2 pl-10">
+              {applicant?.externallyReviewed && (
+                <li key={applicantName} className="mb-4">
+                  {applicantName}
+                </li>
+              )}
+              {children.map((child, index) => {
+                if (child.externallyReviewed) {
+                  const childName = `${child.firstName} ${child.lastName}`;
+                  return (
+                    <li key={childName} className="mb-4">
+                      {childName}
+                    </li>
+                  );
+                }
+                return <></>;
+              })}
```

Comment:
Nitpick: We can use `filter(..)` to exclude the non-reviewed children and avoid the empty fragments `<></>`. We can also remove `index` parameter from the `map(..)` as it is not being used.

```suggestion
              {children
                .filter((child) => child.externallyReviewed)
                .map((child) => {
                  const childName = `${child.firstName} ${child.lastName}`;
                  return (
                    <li key={childName} className="mb-4">
                      {childName}
                    </li>
                  );
                })}
```